src/config/AppConfiguration.js

```diff
@@ -35,3 +35,7 @@ export const ADD_SCENARIO_PARENT_LAST_RUN_ID_PARAMETER = false;
 export const ADD_SCENARIO_MASTER_ID_PARAMETER = false;
 export const ADD_SCENARIO_MASTER_LAST_RUN_ID_PARAMETER = false;
 export const ADD_SCENARIO_RUN_TEMPLATE_NAME_PARAMETER = false;
+
+// Power BI
+// true => Aad false => Embed
```

Comment:
Can we remove the `true => Aad false => Embed` comment (we have doc to explain that) or put it on the same line as the parameter, to decrease the risks of conflicts for solutions in the configuration files ?